x-pack/plugins/data_usage/server/routes/internal/data_streams.ts

```diff
@@ -15,23 +15,21 @@ export const registerDataStreamsRoute = (
   router: DataUsageRouter,
   dataUsageContext: DataUsageContext
 ) => {
-  if (dataUsageContext.serverConfig.enabled) {
```

Comment:
i dont think we need this?  because if the plugin is disabled these wont register?  @ashokaditya

Reply:
Yeah that's sounds right.